### PR TITLE
OBSDOCS-1499 - Logging 5.8.15 RN - OCP 4.12 thru 4.15

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.12, 4.13, 4.14, 4.15

Issue: [OBSDOCS-1499](https://issues.redhat.com//browse/OBSDOCS-1499) 

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by libander** [comment]
@anpingli - Review please.

**Review by anpingli — COMMENTED** [review]
LGTM

**Review by xperimental — APPROVED** [review]

File: modules/logging-release-notes-5-8-15.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-15_{context}"]
+= Logging 5.8.15
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10052[RHBA-2024:10052] and link:https://access.redhat.com/errata/RHBA-2024:140310[RHBA-2024:140310].
+[id="logging-release-notes-5-8-15-bug-fixes_{context}"]
+== Bug fixes
+*  Before this update, Loki did not correctly load some configurations, which caused issues when using Alibaba Cloud or IBM Cloud object storage. This update fixes the configuration-loading code in Loki, resolving the issue. (link:https://issues.redhat.com/browse/LOG-6294[LOG-6294])
+* Before this update, upgrades to version 6.0 failed with errors if a Log File Metric Exporter instance was present. This update fixes the issue, enabling upgrades to proceed smoothly without errors. (link:https://issues.redhat.com/browse/LOG-6328[LOG-6328])
+[id="logging-release-notes-5-8-15-CVEs_{context}"]
+== CVEs
```
Comment (aspauldi) [inline modules/logging-release-notes-5-8-15.adoc:12]: In the preview, **== CVEs** appears to be on the wrong line. It's tucked in after the bug link and should likely be on its own line following the bug link.

File: observability/logging/logging_release_notes/logging-5-8-release-notes.adoc [hunk 2]
```diff
@@ -10,6 +10,8 @@ include::snippets/logging-compatibility-snip.adoc[]
 
 include::snippets/logging-stable-updates-snip.adoc[]
 
+include::modules/logging-release-notes-5-8-15.adoc[leveloffset=+1]
+
 include::modules/logging-release-notes-5-8-14.adoc[leveloffset=+1]
```
Comment (aspauldi) [inline observability/logging/logging_release_notes/logging-5-8-release-notes.adoc:15]: The preview for this module says: "This release includes [OpenShift Logging Bug Fix Release 5.8.14](https://access.redhat.com/errata/RHSA-2024:8317) and [OpenShift Logging Bug Fix Release 5.8.14](https://access.redhat.com/errata/RHBA-2024:8316)." Should one of those be release 5.8.15 or . . . .?

**Review by aspauldi — COMMENTED** [review]
Hi Libby, I noticed just a couple of things.

File: observability/logging/logging_release_notes/logging-5-8-release-notes.adoc (hunk 2 above)
Comment (libander) [reply observability/logging/logging_release_notes/logging-5-8-release-notes.adoc:15]: No, one is for kibana and the other for the CLO, the links go to different errata.

File: modules/logging-release-notes-5-8-15.adoc (hunk 1 above)
Comment (libander) [reply modules/logging-release-notes-5-8-15.adoc:12]: Good eyes, thanks!

**Comment by libander** [comment]
Release still pending.

File: modules/logging-release-notes-5-8-15.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-15_{context}"]
+= Logging 5.8.15
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10052[RHBA-2024:10052] and link:https://access.redhat.com/errata/RHBA-2024:140310[RHBA-2024:140310].
```
Comment (bergerhoffer) [inline modules/logging-release-notes-5-8-15.adoc:6]: These links are still 404ing. Has it not actually been released yet, or is something wrong with the URLs?

**Review by bergerhoffer — COMMENTED** [review]
@libander The advisory links are 404ing. Can you check and see if the release hasn't gone live yet, or if the links are wrong or something?

File: modules/logging-release-notes-5-8-15.adoc (hunk 3 above)
Comment (libander) [reply modules/logging-release-notes-5-8-15.adoc:6]: Something went wrong with the actual release, several times. Still pending.

**Comment by libander** [comment]
Shipped live, should be good to merge. Thanks again @bergerhoffer ! :)

File: modules/logging-release-notes-5-8-15.adoc (hunk 3 above)
Comment (bmcelvee) [reply modules/logging-release-notes-5-8-15.adoc:6]: This one is still 404ing: https://access.redhat.com/errata/RHBA-2024:140310

I'll put it back into the queue until it's ready to go

Comment (adellape) [reply modules/logging-release-notes-5-8-15.adoc:6]: @libander Should this be:
```suggestion
This release includes link:https://access.redhat.com/errata/RHBA-2024:10052[RHBA-2024:10052] and link:https://access.redhat.com/errata/RHBA-2024:10053[RHBA-2024:10053].
```
Looks like it got the new advisory number of `10053` when it shipped, and that one is available/live: https://access.redhat.com/errata/RHBA-2024:10053

**Comment by adellape** [comment]
Holding for https://github.com/openshift/openshift-docs/pull/85216#discussion_r1854265220

**Comment by adellape** [comment]
Dropping from merge queue til the broken link is updated. Can re-queue for next week's merge-review squad.

File: modules/logging-release-notes-5-8-15.adoc (hunk 3 above)
Comment (xperimental) [reply modules/logging-release-notes-5-8-15.adoc:6]: Sorry, I missed the second advisory link during the original review and only checked the first one. 10052 and 10053 are the correct numbers.

**Comment by gabriel-rh** [comment]
closing this on in favour of https://github.com/openshift/openshift-docs/pull/85216

**Pull request closed without merging**